New language swedish

felixdro opened the pull request:
## What does this PR do?
Adding Swedish translations to Coral 5 
New files: src/locales/sv/*, src/core/server/locales/sv/*
Updated files: locales.ts , general.spec.tsx.snap

## How do I test this PR?
Select Swedish

felixdro committed:
Adding Swedish translation

felixdro committed:
updated snap file for Swedish translation

losowsky commented:
Thanks for this, Felix! Is Coral live on computersweden/other IDG sites yet? We'd love to see it working in Swedish?

kgardnr commented on src/locales/sv/auth.ftl:
```diff
@@ -0,0 +1,117 @@
+  To protect against unauthorized changes to your account,
+  we require users to create a password.
+createPassword-createPasswordButton =
+  Create Password
```
Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

felixdro replied:
This file is now 100 % translated

kgardnr commented on src/locales/sv/install.ftl:
```diff
@@ -0,0 +1,63 @@
+
+install-finalStep-description = Thanks for installing { -product-name }! We sent an email to verify your email address. While you finish setting up the account, you can start engaging with your readers now.
+install-finalStep-goToTheDocs = Go to the Docs
+install-finalStep-goToAdmin = Go to Admin
```
Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

felixdro replied:
This file is now partially translated

felixdro replied:
@kgardnr Or is it better to remove the file from the Swedish translation instead of only partially translating it?

felixdro replied:
This file is deleted now

kgardnr commented on src/core/server/locales/sv/errors.ftl:
```diff
@@ -0,0 +1,59 @@
+error-passwordIncorrect = Incorrect password. Please try again.
+error-usernameAlreadyUpdated = You may only change your username once every { framework-timeago-time }.
+error-persistedQueryNotFound = The persisted query with ID { $id } was not found.
+error-rawQueryNotAuthorized = You are not authorized to execute this query.
```
Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

felixdro replied:
This file is now partially translated too.

felixdro replied:
This file is 100 % translated now

felixdro commented:
> Thanks for this, Felix! Is Coral live on computersweden/other IDG sites yet? We'd love to see it working in Swedish?

And thanks for developing Coral! Yes it's live on several of our sites now! We recently released for out members and havn't got that many comments just yet. 

Here is an open article (with one comment currently) https://pcforalla.idg.se/2.1054/1.728097/bluescreen-view#commentStream

felixdro committed:
Translated sv/auth.ftl and partially translated install.ftl and errors.ftl to swedish

kgardnr committed:
Merge branch 'master' into new-language-swedish

kgardnr commented:
Hey @felixdro do you want me to leave this PR open so you can contribute the missing translations?

Also, I can fix those merge conflicts or you can, up to you!

felixdro committed:
Translated server/error completly and removed the swedish translation file install

felixdro committed:
Merge branch 'new-language-swedish' of https://github.com/felixdro/talk into new-language-swedish

felixdro committed:
Fixed merge conflict with Finnish language that was added before this PR was completed

felixdro committed:
Merge branch 'master' into new-language-swedish

felixdro commented:
> Hey @felixdro do you want me to leave this PR open so you can contribute the missing translations?
> 
> Also, I can fix those merge conflicts or you can, up to you!

@kgardnr I deleted the Swedish translation file src/locales/sv/install.ftl and translated the server/error.ftl 100% and pushed those changes.
There was a conflict with the new Finnish translation in two files that I had missed but I solved them too. (src/core/common/helpers/i18n/locales.ts and src/core/client/admin/test/configure/__snapshots__/general.spec.tsx.snap)

 ̶B̶u̶t̶ ̶n̶o̶w̶ ̶t̶h̶e̶ ̶t̶e̶s̶t̶s̶ ̶f̶a̶i̶l̶s̶ ̶a̶n̶d̶ ̶I̶ ̶d̶o̶n̶'̶t̶ ̶k̶n̶o̶w̶ ̶w̶h̶y̶ ̶(̶I̶'̶m̶ ̶a̶ ̶b̶i̶t̶ ̶n̶e̶w̶ ̶t̶o̶ ̶t̶h̶i̶s̶ ̶a̶n̶d̶ ̶h̶o̶p̶e̶ ̶I̶ ̶d̶i̶d̶n̶'̶t̶ ̶d̶o̶ ̶a̶n̶y̶t̶h̶i̶n̶g̶ ̶w̶r̶o̶n̶g̶)̶.̶ ̶T̶e̶l̶l̶ ̶m̶e̶ ̶i̶f̶ ̶I̶ ̶s̶h̶o̶u̶l̶d̶ ̶d̶o̶ ̶a̶n̶y̶t̶h̶i̶n̶g̶ ̶e̶l̶s̶e̶,̶ ̶i̶f̶ ̶t̶h̶e̶r̶e̶ ̶i̶s̶ ̶a̶n̶y̶ ̶m̶o̶r̶e̶ ̶m̶e̶r̶g̶e̶ ̶c̶o̶n̶f̶l̶i̶c̶t̶s̶ ̶I̶ ̶w̶o̶u̶l̶d̶ ̶b̶e̶ ̶g̶l̶a̶d̶ ̶i̶f̶ ̶y̶o̶u̶ ̶c̶a̶n̶ ̶f̶i̶x̶ ̶t̶h̶e̶m̶.̶ ̶T̶h̶a̶n̶k̶s̶!̶

Edit: I fixed the new errors and pushed them here again. Now all the tests pass.

felixdro committed:
Fixed wrong semi colon and white space

kgardnr committed:
Merge branch 'master' into new-language-swedish
